mathesar_ui/src/pages/admin-update/ReleaseBox.svelte

```diff
@@ -0,0 +1,142 @@
+<script lang="ts">
+  import { Button, Icon } from '@mathesar-component-library';
+  import Logo from '@mathesar/components/Logo.svelte';
+  import {
+    iconCurrentlyInstalledVersion,
+    iconExternalHyperlink,
+    iconUpgradeAvailable,
+  } from '@mathesar/icons';
+  import { modal } from '@mathesar/stores/modal';
+  import type { Release } from '@mathesar/stores/releases';
+  import { assertExhaustive } from '@mathesar/utils/typeUtils';
+  import UpgradeModal from './upgrade-modal/UpgradeModal.svelte';
+
+  const modalController = modal.spawnModalController();
+
+  export let type:
+    | 'available-upgrade'
+    | 'currently-installed-and-latest'
+    | 'current'
+    | 'latest';
+  export let release: Release;
+
+  $: date = new Date(release.date);
+  $: dateString = date.toLocaleDateString();
+</script>
+
+<div
+  class="release"
+  class:available-upgrade={type === 'available-upgrade'}
+  class:currently-installed-and-latest={type ===
+    'currently-installed-and-latest'}
+>
+  <div class="type">
+    {#if type === 'available-upgrade'}
+      <Icon {...iconUpgradeAvailable} />
+      New Version Available
+    {:else if type === 'currently-installed-and-latest'}
+      <Icon {...iconCurrentlyInstalledVersion} />
+      You are running the latest version
+    {:else if type === 'current'}
+      <Icon {...iconCurrentlyInstalledVersion} />
+      Currently Installed
+    {:else if type === 'latest'}
+      Latest Available Version (not installed)
+    {:else}
+      {assertExhaustive(type)}
+    {/if}
+  </div>
+  <div class="details">
+    <div class="left">
+      <div class="logo"><Logo /></div>
+      <div class="name">Mathesar</div>
+      <div class="version">{release.tagName}</div>
+    </div>
+    <div class="right">
+      <div class="date">Released {dateString}</div>
+      <a href={release.notesUrl} class="notes">
+        Release Notes <Icon {...iconExternalHyperlink} />
+      </a>
```

Comment:
I think it'd be better if we open this in a new browser tab. The icon also indicates that this opens in an external tab.

Reply:
Fixed in a6fd82052517802baffb6f784da4e607323d57da.